### PR TITLE
Grammar update in README.md

### PR DESCRIPTION
**Pull request by aryman1503**
Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use submitGit to conveniently send your Pull
Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

File: git-gui/po/README [hunk 1]
```diff
@@ -4,7 +4,7 @@ Localizing git-gui for your language
 This short note is to help you, who reads and writes English and your
 own language, help us getting git-gui localized for more languages.  It
 does not try to be a comprehensive manual of GNU gettext, which is the
-i18n framework we use, but tries to help you get started by covering the
+i18n framework we use but tries to help you get started by covering them
```
Comment (dscho) [inline git-gui/po/README]: I am no expert about using commas in English, but "them basics" sounds very colloquial to me.

File: git-gui/po/README [hunk 2]
```diff
@@ -21,9 +21,9 @@ them.
 You would then need to clone the git-gui project repository and create
 a feature branch to begin working:
 
-	$ git clone git://repo.or.cz/git-gui.git
-	$ cd git-gui.git
-	$ git checkout -b my-translation
+    $ git clone git://repo.or.cz/git-gui.git
+    $ cd git-gui.git
+    $ git checkout -b my-translation
```
Comment (dscho) [inline git-gui/po/README]: This ain't no grammar fix.

File: git-gui/po/README [hunk 3]
```diff
@@ -37,9 +37,9 @@ files whose names end with ".po".  Is there a file that has messages
 in your language?
 
 If you do not know what your language should be named, you need to find
-it.  This currently follows ISO 639-1 two letter codes:
+it.  This currently follows ISO 639-1 two-letter codes:
```
Comment (dscho) [inline git-gui/po/README]: We recently had a pretty disruptive change from "up-to-date" to "up to date". In this light, I don't think that "two-letter" is an improvement over "two letter".

File: git-gui/po/README [hunk 4]
```diff
@@ -37,9 +37,9 @@ files whose names end with ".po".  Is there a file that has messages
...
 
-	http://www.loc.gov/standards/iso639-2/php/code_list.php
+    http://www.loc.gov/standards/iso639-2/php/code_list.php
```
Comment (dscho) [inline git-gui/po/README]: Again, this changes whitespace, not grammar.

File: git-gui/po/README [hunk 5]
```diff
@@ -73,19 +73,18 @@ look something like this:
 
 You will find many pairs of a "msgid" line followed by a "msgstr" line.
 These pairs define how messages in git-gui application are translated to
-your language.  Your primarily job is to fill in the empty double quote
+your language.  Your primary job is to fill in the empty double quote
```
Comment (dscho) [inline git-gui/po/README]: This is the first fix in this PR that I agree with 100%.

File: git-gui/po/README [hunk 6]
```diff
@@ -73,19 +73,18 @@ look something like this:
...
 pairs on msgstr lines with the translation of the strings on their
 matching msgid lines.  A few tips:
 
  - Control characters, such as newlines, are written in backslash
-   sequence similar to string literals in the C programming language.
+   a sequence similar to string literals in the C programming language.
```
Comment (dscho) [inline git-gui/po/README]: Did you mean to insert the "a" before "backslash" instead of after it?

File: git-gui/po/README [hunk 7]
```diff
@@ -73,19 +73,18 @@ look something like this:
...
    When the string given on a msgid line has such a backslash sequence,
    you would typically want to have corresponding ones in the string on
    your msgstr line.
 
  - Some messages contain an optional context indicator at the end,
-   for example "@@noun" or "@@verb".  This indicator allows the
-   software to select the correct translation depending upon the use.
+   for example, "@@noun" or "@@verb".  This indicator allows the software to select the correct translation depending upon the use.
```
Comment (dscho) [inline git-gui/po/README]: This not only does not fix any grammar, but it breaks the wrapping.

File: git-gui/po/README [hunk 8]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
 
    When you have to change the order of words, you can add "<number>$"
    between '%' and the conversion ('s', 'd', etc.) to say "<number>-th
-   parameter to the format string is used at this point".  For example,
+   a parameter to the format string is used at this point".  For example,
```
Comment (dscho) [inline git-gui/po/README]: Did you mean to insert the "a" *before* the "<number>-th" (as opposed to appending it)?

File: git-gui/po/README [hunk 9]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
    if the original message is like this:
 
-	"Length is %d, Weight is %d"
+    "Length is %d, Weight is %d"
```
Comment (dscho) [inline git-gui/po/README]: This is a whitespace change that might, or might not be desirable. My guess is that it is not wanted.

File: git-gui/po/README [hunk 10]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
 
    and if for whatever reason your translation needs to say weight first
    and then length, you can say something like:
 
-	"WEIGHT IS %2$d, LENGTH IS %1$d"
+    "WEIGHT IS %2$d, LENGTH IS %1$d"
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 11]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
 
-   A format specification with a '*' (asterisk) refers to *two* arguments
-   instead of one, hence the succeeding argument number is two higher
-   instead of one. So, a message like this
+   A format specification with a '*' (asterisk) refers to *two* arguments instead of one, hence the succeeding argument number is two higher instead of one. So, a message like this
 
-	"%s ... %*i of %*i %s (%3i%%)"
+    "%s ... %*i of %*i %s (%3i%%)"
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 12]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
 
    is equivalent to
 
-	"%1$s ... %2$*i of %4$*i %6$s (%7$3i%%)"
+    "%1$s ... %2$*i of %4$*i %6$s (%7$3i%%)"
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 13]
```diff
@@ -136,26 +131,26 @@ You can test your translation by running "make install", which would
 create po/af.msg file and installs the result, and then running the
 resulting git-gui under your locale:
 
-	$ make install
-	$ LANG=af git-gui
+    $ make install
+    $ LANG=af git-gui
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 14]
```diff
@@ -136,26 +131,26 @@ You can test your translation by running "make install", which would
...
 
 There is a trick to test your translation without first installing:
 
-	$ make
-	$ LANG=af ./git-gui.sh
+    $ make
+    $ LANG=af ./git-gui.sh
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 15]
```diff
@@ -136,26 +131,26 @@ You can test your translation by running "make install", which would
...
-	   master..
+    $ edit po/af.po
+    ... be sure to update Last-Translator: and
+    ... PO-Revision-Date: lines.
+    $ git add po/af.po
+    $ git commit -s -m 'git-gui: added Afrikaans translation.'
+    $ git send-email --to 'git@vger.kernel.org' \
+       --cc 'Pat Thoyts <patthoyts@users.sourceforge.net>' \
+       --subject 'git-gui: Afrikaans translation' \
+       master..
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 16]
```diff
@@ -167,8 +162,8 @@ itself was updated and there are new messages that need translation.
 
 In any case, make sure you are up to date before starting your work:
 
-	$ git checkout master
-	$ git pull
+    $ git checkout master
+    $ git pull
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 17]
```diff
@@ -180,7 +175,7 @@ po/af.po file, you can tell if there are new messages that need to be
 translated.  You would need the GNU gettext package to perform this
 step.
 
-	$ msgmerge -U po/af.po po/git-gui.pot
+    $ msgmerge -U po/af.po po/git-gui.pot
```
Comment (dscho) [inline git-gui/po/README]: Whitespace change... please revert.

File: git-gui/po/README [hunk 18]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
    and if for whatever reason your translation needs to say weight first
    and then length, you can say something like:
 
-	"WEIGHT IS %2$d, LENGTH IS %1$d"
+    "WEIGHT IS %2$d, LENGTH IS %1$d"
 
-   A format specification with a '*' (asterisk) refers to *two* arguments
-   instead of one, hence the succeeding argument number is two higher
-   instead of one. So, a message like this
+   A format specification with a '*' (asterisk) refers to *two* arguments instead of one, hence the succeeding argument number is two higher instead of one. So, a message like this
```
Comment (dscho) [inline git-gui/po/README]: This breaks the formerly correct formatting. Please revert.

File: git-gui/po/README [hunk 19]
```diff
@@ -98,30 +97,26 @@ matching msgid lines.  A few tips:
...
    is equivalent to
 
-	"%1$s ... %2$*i of %4$*i %6$s (%7$3i%%)"
+    "%1$s ... %2$*i of %4$*i %6$s (%7$3i%%)"
 
  - A long message can be split across multiple lines by ending the
-   string with a double quote, and starting another string on the next
-   line with another double quote.  They will be concatenated in the
-   result.  For example:
+   string with a double quote, and starting another string on the next line with another double quote.  They will be concatenated in the result.  For example:
```
Comment (dscho) [inline git-gui/po/README]: This breaks the formerly correct formatting. Please revert.

**Comment by dscho** [comment]
@aryman1503 are you still interested in this PR?

**Comment by dscho** [comment]
@aryman1503 gentle ping?

**Comment by aryman1503** [comment]
Hi,

I am not getting a proper time but I am interested to be a part of an
open-source project that's why I am trying to contribute in this way.
Give me some time I will fix all the bug which you mention and If you think
that no need to change in files then it is okay. And thank you for letting
me know it will help me in my future contribution.

Regards,
Aryman

**Comment by dscho** [comment]
> Give me some time I will fix all the bug which you mention

Of course!

> and If you think that no need to change in files then it is okay.

Oh, I think that some of the changes should be submitted to the Git project.

You might want to follow the advice in https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits to break the commit apart into individual changes, and drop the ones that I pointed out should be reverted.

**Comment by aryman1503** [comment]
Sure I will do the same.

**Comment by dscho** [comment]
> Sure I will do the same.

It's been more than a month and a half... any progress?

**Comment by aryman1503** [comment]
No sir, actually I am busy to finding a job for me. I have done my
engineering this year so I am busy for the same.

**Comment by dscho** [comment]
> right now you can close this PR.

So can you ;-) I do not necessarily like to "play cleaner".

**Comment by aryman1503** [comment]
Thank you for your time and consideration, and sorry that I am not
continuing my PR. Hoping in future I will be part of this.

**Pull request closed without merging**